content/docs/rfcs/44/README.md

```diff
@@ -0,0 +1,272 @@
+
+Note: This is an early raw version of the specification.
+It does not strictly follow the formally evaluated Dandelion++ paper,
+as we want to experiment with relaxing (and strengthening) certain model properties.
+In specific, we aim at a version that has tighter latency bounds.
+[Further research](https://arxiv.org/pdf/2201.11860.pdf) suggests that Dandelion++'s design choices are not optimal,
+which further assures that tweaking design choices makes sense.
+We will refine design decisions in future versions of this specification.
+
+Further information on Waku anonymity may be found in our [Waku Privacy and Anonymity Analysis](https://vac.dev/wakuv2-relay-anon).
+
+# Theory and Functioning
+
+44/WAKU2-DANDELION can be seen as an anonymity enhancing add-on to [Waku Relay](https://rfc.vac.dev/spec/11/) message dissemination,
+which is based on [libp2p gossipsub](https://github.com/libp2p/specs/blob/master/pubsub/gossipsub/README.md).
+44/WAKU2-DANDELION subdivides message dissemination into a "stem" and a "fluff" phase.
+This specification is mainly concerned with specifying the stem phase.
+The fluff phase corresponds to [Waku Relay](/spec/11/),
+except that it adds a random delay before relaying a message.
+
+*Note: In a future version of this specification,
+we might move the specification of the fluff phase into a separate document.
+We envision stem and fluff phase as abstract concepts.
+The Dandelion stem and fluff phases instantiate these concepts.
+Future stem specifications might comprise: none (standard relay), Dandelion, Tor, and mix-net.
+As for future fluff specifications: none (standard relay), diffusion (random delays), and mix-net.*
+
+Messages relayed by nodes supporting 44/WAKU2-DANDELION are either in stem phase or in fluff phase.
+We refer to the former as a stem message and to the latter as a fluff message.
+A message starts in stem phase, and at some point, transitions to fluff phase.
+Nodes, on the other hand, are in stem state or fluff state.
+Nodes in stem state relay stem messages to a single relay node, randomly selected per epoch for each incoming stem connection.
+Nodes in fluff state transition stem messages into fluff phase and relay them accordingly.
+Fluff messages are always disseminated via Waku Relay (with added random delay).
+
+Messages originated in the node (i.e. messages coming from the application layer of our node),
+are always sent as stem messages.
+
+The stem phase can be seen as a different protocol, and messages are introduced into Waku Relay, and by extension gossipsub,
+once they arrive at a node in fluff state for the first time.
+44/WAKU2-DANDELION uses [19/WAKU2-LIGHTPUSH](/spec/19/) as the protocol for relaying stem messages.
+
+There are no negative effects on gossipsub peer scoring,
+because Dandelion nodes in *stem state* still normally relay Waku Relay (gossipsub) messages.
+
+# Specification
+
+Nodes supporting 44/WAKU2-DANDELION MUST either be in stem state or in fluff state.
```

Comment:
Nodes may have different states for different messages, right? So it is not a static state per node?